packages/numbers/tests/rest/v1/available-number/available-number.steps.ts

```diff
@@ -0,0 +1,153 @@
+import { AvailableNumberApi, NumbersService, Numbers } from '../../../../src';
+import { Given, Then, When } from '@cucumber/cucumber';
+import assert from 'assert';
+
+let availableNumberApi: AvailableNumberApi;
+let availableNumbersResponse: Numbers.AvailableNumbersResponse;
+let availablePhoneNumber: Numbers.AvailableNumber;
+let activeNumber: Numbers.ActiveNumber;
+
+Given('the Numbers service "Available Number" is available', function () {
+  const numbersService = new NumbersService({
+    projectId: 'tinyfrog-jump-high-over-lilypadbasin',
+    keyId: 'keyId',
+    keySecret: 'keySecret',
+    authHostname: 'http://localhost:3011',
+    numbersHostname: 'http://localhost:3013',
+  });
+  availableNumberApi = numbersService.availableNumber;
+});
+
+When('I send a request to list the available phone numbers', async () => {
+  availableNumbersResponse = await availableNumberApi.list({
+    regionCode: 'US',
+    type: 'LOCAL',
+  });
+});
+
+Then('the response contains {string} available phone numbers', (expectedAnswer: string) => {
+  const expectedPhoneNumbersCount = parseInt(expectedAnswer, 10);
+  assert.equal(availableNumbersResponse.availableNumbers?.length, expectedPhoneNumbersCount);
+});
+
+Then('a phone number contains all the expected properties', () => {
+  const phoneNumber = availableNumbersResponse.availableNumbers![0];
+  assert.equal(phoneNumber.phoneNumber, '+12013504948');
+  assert.equal(phoneNumber.regionCode, 'US');
+  assert.equal(phoneNumber.type, 'LOCAL');
+  assert.deepEqual(phoneNumber.capability, ['SMS', 'VOICE']);
+  assert.deepEqual(phoneNumber.setupPrice, { currencyCode: 'EUR', amount: '0.80' });
```

Comment:
unrelated to this PR, but may be for SDK's users facility, amount could have been transformed to a Number in place of String